zebedee-cms/src/test/java/com/github/onsdigital/zebedee/ZebedeeTest.java

```diff
@@ -18,200 +20,408 @@
+        assertEquals(expectedURI, completeURI);
+        assertEquals(expectedURI, reviewedURI);
+    }
+
+    @Test
+    public void toUri_givenPathOutsideZebedee_shouldReturnNull() throws IOException {
+        // Given
+        // a zebedee implementation
+        Zebedee zebedee = new Zebedee(new ZebedeeConfiguration(expectedPath, false));
+        String notZebedee = "/NotZebedee/data.json"; // (non zebedee path)
+        Path notZebedeePath = Paths.get(notZebedee);
+
+        // When
+        // we convert these to URIs
+        String notZebedeeUri = zebedee.toUri(notZebedeePath);
+
+        // Then
+        // we expect the uri to be null
+        assertNull(notZebedeeUri);
+    }
+
+    @Test
+    public void shouldReturnCollectionThatContainsSpecifiedURIIfExists() throws IOException, CollectionNotFoundException {
+        Zebedee zebedee = new Zebedee(new ZebedeeConfiguration(expectedPath, false));
+        Collection collectionOne = zebedee.getCollections().getCollectionByName(COLLECTION_ONE_NAME);
+        Collection collectionTwo = zebedee.getCollections().getCollectionByName(COLLECTION_TWO_NAME);
+
+        String contentPath = "/aboutus/data.json";
+
+        // create content in collection 01.
+        builder.createInProgressFile(contentPath);
+
+        Session session = new Session();
+        session.setEmail("makingData@greatagain.com");
+        Optional<Collection> blockingCollection = zebedee.checkForCollectionBlockingChange(collectionOne, contentPath);
+
+        assertThat(blockingCollection.isPresent(), is(true));
+        assertThat(blockingCollection.get().getDescription().getName(), equalTo(collectionTwo.getDescription().getName()));
+        assertThat(collectionTwo.inProgressUris().contains(contentPath), is(true));
+        assertThat(collectionTwo.completeUris().contains(contentPath), is(false));
+        assertThat(collectionTwo.reviewedUris().contains(contentPath), is(false));
+    }
+
+    @Test
+    public void shouldReturnEmptyOptionalIfNoCollectionContainsSpecifiedURI() throws IOException, CollectionNotFoundException {
+        Zebedee zebedee = new Zebedee(new ZebedeeConfiguration(expectedPath, false));
+        String contentPath = "/aboutus/data.json";
+        Collection collectionOne = zebedee.getCollections().getCollectionByName(COLLECTION_ONE_NAME);
+
+        Session session = new Session();
+        session.setEmail("makingData@greatagain.com");
+        Optional<Collection> blockingCollection = zebedee.checkForCollectionBlockingChange(collectionOne, contentPath);
+        assertThat(blockingCollection.isPresent(), is(false));
+
+        Collection collectionTwo = zebedee.getCollections().getCollectionByName(COLLECTION_TWO_NAME);
+        assertThat(collectionTwo.inProgressUris().contains(contentPath), is(false));
+        assertThat(collectionTwo.completeUris().contains(contentPath), is(false));
+        assertThat(collectionTwo.reviewedUris().contains(contentPath), is(false));
+    }
```

Comment:
Only change here is update to formatting (replaced tabs with spaces).

New code below 👇